Tests/FunctionalTests/Sources/FTRKeyboardKeysTest.m

```diff
@@ -464,6 +464,20 @@ - (void)testTypingOnTextFieldInUIInputAccessory {
       performAction:grey_typeText(@"Test")] assertWithMatcher:grey_text(@"Test")];
 }
 
+- (void)testTypingOnUIAccessibilityTextFieldElement {
+  [[EarlGrey selectElementWithMatcher:grey_accessibilityID(@"Input Button")]
+      performAction:grey_tap()];
+  Class accessibilityTextFieldElemClass = NSClassFromString(@"UIAccessibilityTextFieldElement");
+  [[EarlGrey selectElementWithMatcher:grey_allOf(grey_accessibilityValue(@"Text Field"),
+                                                 grey_kindOfClass(accessibilityTextFieldElemClass),
+                                                 nil)]
+      performAction:grey_clearText()];
+  [[EarlGrey selectElementWithMatcher:grey_allOf(grey_accessibilityValue(@"Text Field"),
+                                                 grey_kindOfClass(accessibilityTextFieldElemClass),
+                                                 nil)]
+      performAction:grey_replaceText(@"")];
```

Comment:
In that case I suggest renaming the test to something like `testClearAndReplaceWorksWithAccessibilityElements`